frontend/src/views/NavigationDefault.vue

```diff
@@ -6,21 +6,33 @@
     color="blue-grey darken-4"
     dark
   >
-    <logo text />
+    <Logo text />
     <v-spacer />
-    <user-meta />
+    <v-toolbar-items v-if="$vuetify.breakpoint.lgAndUp">
```

Comment:
you need to hide the button if the link is not configured (if we want to enable a not configured value)

Reply:
Could you help me with the helm config?

Reply:
The termsOfServiceLinkTemplate has the same problem, and @manuelmeister made it consistent.
I opened another issue to fix this: https://github.com/ecamp/ecamp3/issues/4777